.npmignore

```diff
@@ -79,3 +79,7 @@ test
 
 # release scripts
 script
+
+# FFI binaries
+ffi
```

Comment:
Is this right? Wouldn't the new binaries need the libraries around, or are they statically linked?

Also, wouldn't the `ffi` directory still be needed for archs where there's no prebuild?

Reply:
binaries get moved into the prebuild directory where there will be an platform-arch folder.

the changes in the node gyp file will conditionally copy on the relevant shared library for the architecture/platform.

as we only supply 5 shared libs for pact ffi and will pre build those combinations, there won’t be any ability for a user to build for any other arch/plat combos and therefore no requirement to duplicate any binaries or files in the ffi directory when published to npm, all the binaries will be packed in the relevant arch specific pre build folder